airbyte-integrations/connectors/source-tplcentral/integration_tests/sample_state.json

```diff
@@ -0,0 +1,5 @@
+{
+  "todo-stream-name": {
+    "todo-field-name": "value"
+  }
+}
```

Comment:
remove this if this is not needed, if needed can you change this to actual stream name

Reply:
Fixed.